neural_compressor/adaptor/torch_utils/hawq_metric.py

```diff
@@ -21,13 +21,17 @@
 
 import copy
 import numpy as np
+from collections import OrderedDict
 import torch.nn
 from torch.quantization.quantize_fx import fuse_fx
+import torch.nn.intrinsic.quantized as nniq
+from torch.fx import symbolic_trace, graph_module
```

Comment:
please remove the import not be used

Reply:
OK, i removed them and pls review it again